app/services/item_create_service.rb

```diff
@@ -1,4 +1,7 @@
 class ItemCreateService
+  Success = Data.define(:item) { def success? = true }
+  Failure = Data.define(:error) { def success? = false }
```

Comment:
Wouldn't you just have a single `Result` class that can have either an item or error?

Reply:
Then you could define it as part of the general data models along with Option.

Reply:
True, but won't successful results need different attributes for different services? The ItemCreateService returns an item, but the TransferDestroyService doesn't need to return anything.

Reply:
What do you think of the refactor in [4bab8a4](https://github.com/rubyforgood/human-essentials/pull/4832/commits/4bab8a48ba5fd99a67da8e88425be495ce537734)?

Reply:
I generally see this as `Result` that has either `value` or `error`. If you want to know if it was successful or not, just check to see if the error is null. This is Ruby, `value` can be any type :) I don't think we need multiple types.

Reply:
Fixed.